examples/alerts/alerts.md

```diff
@@ -562,6 +562,23 @@ rules:
   for: 3h
   labels:
     severity: critical
+- alert: ThanosReceiveTrafficBelowThreshold
+  annotations:
+    description: At Thanos Receive {{$labels.job}} in {{$labels.namespace}} , the
+      average 1-hr avg. metrics ingestion rate  is {{$value | humanize}}% of 12-hr
+      avg. ingestion rate.
+    runbook_url: https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md#alert-name-thanosreceivetrafficbelowthreshold
+    summary: Thanos Receive is experiencing low avg. 1-hr ingestion rate relative
+      to avg. 12-hr ingestion rate.
+  expr: |
+    (
+      avg by (job) (rate(http_requests_total{code=~"2..", job=~".*thanos-receive.*", handler="receive"}[1h]))
```

Comment:
For average within one hour we need to use sum sub query so something like this:
`avg_over_time(rate(http_requests_total{code=~"2..", job=~".*thanos-receive.*", handler="receive"}[5m])[1h:5m])` 

The `avg` makes average over series not over time.

Reply:
👍  Thanks @bwplotka !